plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/config/Config.java

```diff
@@ -252,6 +260,11 @@ public Builder withExportDatatables(boolean exportDatatables) {
             return this;
         }
 
+        public Builder withIgnoreJdk7Error(boolean ignoreJdk7Error) {
```

Comment:
I think this flag becomes obsolete if by default we try to fix precondition error